stable/datadog/values.yaml

```
@@ -189,6 +189,29 @@ datadog:
   #
   containerLogsPath: /var/lib/docker/containers
 
+  ## @param customLogFilesEnabled - boolean - optional - default: false
+  ## Enable this to define host logs to mount into the container.
+  ## ref: https://docs.datadoghq.com/agent/logs/?tab=tailexistingfiles#custom-log-collection
+  #
+  # customLogFilesEnabled: false
+
+  ## @param customLogFiles - array - optional - default: []
```

As it's an optional array this is implied, we can drop the default part as it doesn't bring much:

```suggestion
  ## @param customLogFiles - array - optional
```